bundles/org.openhab.binding.growatt/README.md

```diff
@@ -133,6 +134,14 @@ The list of all possible channels is as follows:
 | rac                           | Number:Power              | Reactive 'power' (var).                              | yes      |
 | erac-today                    | Number:Energy             | Reactive 'energy' today (kvarh).                     | yes      |
 | erac-total                    | Number:Energy             | Total reactive 'energy' (kvarh).                     | yes      |
+| charge-va                     | Number:Power              | Charging reactive power.                             | yes      |
+| bat-discharge-va              | Number:Power              | Battery discharging reactive power.                  | yes      |
+| bat-discharge-watt            | Number:Power              | Battery discharging power.                           | yes      |
+| bat-discharge-energy-today    | Number:Energy             | Battery discharge energy today.                      | yes      |
+| bat-discharge-energy-total    | Number:Energy             | Total battery discharge energy.                      | yes      |
```

Comment:
Is there a reason for truncating these channel id prefixes to "bat-" when the other existing battery channels carry the full prefix "battery-"? Also, assuming they are related to the same battery, would it make sense to add them in the table right after the others (i.e. after line 121)?